sourcegraph/about
# Client logo updates

Closes #5404 by removing the PayPal logo and re-ordering 2 logo spots.

### Test

1. Ensure prettier has standardized the proposed changes.
2. Ensure `CustomerLogos` is reflecting the accurate logo specs
